server/src/main/java/org/opensearch/action/support/clustermanager/TransportClusterManagerNodeAction.java

```diff
@@ -117,13 +117,24 @@ protected TransportClusterManagerNodeAction(
 
     protected abstract Response read(StreamInput in) throws IOException;
 
-    protected abstract void masterOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception;
+    /**
+     * @deprecated As of 2.1, because supporting inclusive language, replaced by {@link #clusterManagerOperation(ClusterManagerNodeRequest, ClusterState, ActionListener)}
+     */
+    @Deprecated
+    protected void masterOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
+        throw new UnsupportedOperationException("Must be overridden");
+    }
+
+    protected void clusterManagerOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
+        masterOperation(request, state, listener);
+    }
 
     /**
      * Override this operation if access to the task parameter is needed
      */
-    protected void masterOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
```

Comment:
I think we have to add back the protect method: `protected void masterOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener)` and deprecate it, instead of renaming directly.
The protected method can be accessed through all the subclass of this class `TransportClusterManagerNodeAction`, so we have to add the original method name back to keep backwards compatibility.

Adding the following method in this class:
```
/** @deprecated As of 2.2, because supporting inclusive language, replaced by {@link #clusterManagerOperation(...} */
    protected void masterOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener) throws Exception {
        clusterManagerOperation(task, request, state, listener);
    }
```

Reply:
Got it sure. I will add these changes in next commit.

Reply:
Thanks, I saw it has been updated in the commit https://github.com/opensearch-project/OpenSearch/pull/4032/commits/0a806df0e5fa3a9a3d7eeb2b4e8aab392f1ef54d